sdk/security/keyvault/azsecrets/utils_test.go

```diff
@@ -28,7 +27,7 @@ import (
 )
 
 const recordingDirectory = "sdk/security/keyvault/azsecrets/testdata"
-const fakeVaultURL = "https://fakevault.local"
+const fakeVaultURL = "https://test.vault.azure.net/"
```

Comment:
The proxy now includes a [URL sanitizer](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs#L681). If you followed its pattern, you wouldn't have to add your own sanitizer for URLs.
```suggestion
const fakeVaultURL = fmt.Sprintf("https://%s.vault.azure.net/", recording.SanitizedValue)
```
There's not much advantage in this now but it will save having to record the tests again after deleting `getEnvVar` (which will soon be feasible). Of course you have to record the tests again either way, so it's reasonable not to make this change now.